buildSrc/src/main/java/saros/gradle/intellij/IntellijConfigurator.java

```diff
@@ -58,31 +64,36 @@ private void configureDefaultParams(
     intellijExtension.setUpdateSinceUntilBuild(false);
   }
 
+  private void throwGradleException(String message) {
+    throw new GradleException(message + RACE_CONDITION_WARNING);
+  }
+
   /**
    * Add tasks that locks the runIde sandbox when the runIde task is started and unlocks the sandbox
    * when runIde is finished.
    */
   private void configureLockFileActions(File sandboxDir) {
     Task runIdeTask = project.getTasks().findByPath("runIde");
     if (runIdeTask == null)
-      throw new GradleException("Unable to find the task 'runIde' of the intellij plugin");
+      throwGradleException("Unable to find the task 'runIde' of the intellij plugin.");
 
     runIdeTask.doFirst(
         (Task t) -> {
           try {
             if (!sandboxDir.exists() && !sandboxDir.mkdirs()) {
-              throw new GradleException(
+              throwGradleException(
                   "Unable to create the sandbox directory: " + sandboxDir.getAbsolutePath());
             }
 
             File lockFile = getLockFile(sandboxDir);
             boolean createdNewFile = lockFile.createNewFile();
 
             if (!createdNewFile)
-              throw new GradleException(
-                  "Lock file " + lockFile.getAbsolutePath() + " already exists!");
+              throwGradleException("Lock file " + lockFile.getAbsolutePath() + " already exists!");
           } catch (IOException e) {
-            throw new GradleException("Unable to lock sandbox: " + sandboxDir.getAbsolutePath(), e);
+            throw new GradleException(
+                "Unable to lock sandbox: " + sandboxDir.getAbsolutePath() + RACE_CONDITION_WARNING,
```

Comment:
Add an exception as an optional parameter for `throwGradleException(...)` so that you can also use the method for this case?

Reply:
Added an overloaded method. I just thought adding an extra method for only one case is not necessary. Unfortunately, Java has no optional params.

Reply:
I meant "optional" more in the way of "allow `null` and don't pass the exception if `null`.

Also, remove additional `RACE_CONDITION_WARNING`.
```suggestion
                "Unable to lock sandbox: " + sandboxDir.getAbsolutePath(),
```